frontend/src/components/card/PlayerResultsCard.tsx

```diff
@@ -48,7 +48,10 @@ function PlayerResultsCard(props: PlayerResultsCardProps) {
       <FlexHorizontalContainer>
         <FlexLeft>
           <CenteredContainer>
-            <PlayerIcon hexColor={color.hexColor} />
+            <PlayerIcon
+              hexColor={color.hexColor}
+              active={Boolean(player.user.sessionId || isCurrentPlayer)}
```

Comment:
Just wanted to note that here the current player is always shown to themselves as active, but on the lobby page the current user can be inactive; not sure which should change, both are fine by me, but should those be the same?

Reply:
I'd be down for that - I made this one show up as always active if it's the current player to help minimize potential bugs, so maybe we could do the same for the lobby page. However, maybe we could also wait till the UI PRs for that, since the way the lobby page is currently implemented, it splits it by users in the activeUsers and inactiveUsers list, so doing a conditional check for isCurrentPlayer might not fit into place under the current design.

Reply:
That's fine - I'll get rid of the extra check